src/main/java/org/fisco/bcos/sdk/channel/ChannelMsgHandler.java

```diff
@@ -59,18 +83,14 @@ public void addSeq2CallBack(String seq, ResponseCallback callback) {
 
     @Override
     public void onConnect(ChannelHandlerContext ctx) {
-        // TODO:
-        // queryNodeVersion
-        // queryBlockNumber
-        // connection info -> available peers
+        queryNodeVersion(ctx);
         for (MsgHandler handle : msgConnectHandlerList) {
             handle.onConnect(ctx);
         }
     }
 
     @Override
     public void onMessage(ChannelHandlerContext ctx, Message msg) {
-        // TODO: use msgHandlers to find special type to handle
         ResponseCallback callback = (ResponseCallback) seq2Callback.get(msg.getSeq());
```

Comment:
add getAndRemove(String seq) method for ResponseCallback  object

Reply:
Done